src/proxy.js

```diff
@@ -161,7 +170,7 @@ module.exports = (proxyTarget, host) => {
 						});
 				},
 			};
-		})), true),
+		})), false),
```

Comment:
We certainly can't run this on everything, only HTML + XML.

Reply:
true, I'll take it

Reply:
👍 we can probably keep it at `true` and do our own filtering based on response headers if there's no suitable option in the package directly

Reply:
Here is the PR with a slightly another solution please check if it suits us. Seems to work as expected.
https://github.com/jsdelivr/www.jsdelivr.com/pull/553